src/EnergyPlus/ResultsFramework.hh

```diff
@@ -234,11 +234,17 @@ namespace ResultsFramework {
 
         void writeReport(JsonOutputFilePaths &jsonOutputFilePaths, bool outputJSON, bool outputCBOR, bool outputMsgPack);
 
+        // Need to find a way to protect these, they can't be changed on the fly
+        bool iso8601 = false;
+        bool beginningOfInterval = false;
+
     protected:
         bool IDataFrameEnabled = false;
         bool RDataFrameEnabled = false;
         bool RVariablesScanned = false;
         bool IVariablesScanned = false;
+        int lastHour = 0;
+        int lastMinute = 0;
```

Comment:
These two integers hold the values from the previous call. It would be better if we were storing the whole timestamp here, but there's a bit of a data flow problem (particularly with the year) and getting all that in here anywhere close to initialization time would make this much more complicated.